libass/ass_render.c

```diff
@@ -553,17 +558,17 @@ static bool quantize_transform(double m[3][3], ASS_Vector *pos,
         }
 
     // x_lim = |m_xx| * dx + |m_xy| * dy
-    //  ~= |qm_xx| * q_x * dx + |qm_xy| * q_y * dy
+    //  ≈ |qm_xx| * q_x * dx + |qm_xy| * q_y * dy
     //  = (|qm_xx| + |qm_xy|) * POSITION_PRECISION * z0,
     // y_lim = |m_yx| * dx + |m_yy| * dy
-    //  ~= |qm_yx| * q_x * dx + |qm_yy| * q_y * dy
+    //  ≈ |qm_yx| * q_x * dx + |qm_yy| * q_y * dy
     //  = (|qm_yx| + |qm_yy|) * POSITION_PRECISION * z0,
-    // max(x_lim, y_lim) / z0 ~= w
+    // max(x_lim, y_lim) / z0 ≈ w
     //  = max(|qm_xx| + |qm_xy|, |qm_yx| + |qm_yy|) * POSITION_PRECISION.
 
-    // Quantization steps (ACCURACY ~ 2 * POSITION_PRECISION):
-    // D(m_zx) ~ POSITION_PRECISION * z0^2 / max(x_lim, y_lim) / dx ~= q_zx = q_x / w,
-    // D(m_zy) ~ POSITION_PRECISION * z0^2 / max(x_lim, y_lim) / dy ~= q_zy = q_y / w,
+    // Quantization steps (reminder: ACCURACY ~ 2 * POSITION_PRECISION):
```

Comment:
Similarly, following expressions derived by substituting `ACCURACY` with `2 * POSITION_PRECISION`, it's not a reminder.